### PR TITLE
doc: add machine disruption budget

### PR DESCRIPTION
**Pull request by cynepco3hahue**
`MachineDisruptionBudget` design document.
I want to get some first feedback before I will create the PR under the k8s KPE, and it also nice to have documentation together with code, because of it easier to update documentation if it was some code changes.

### CODE REVIEW

**Comment by cynepco3hahue** [comment]
@ingvagabund @enxebre @beekhof Please review.

File: docs/machine-disruption-budget-kpe.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Machine Disruption Budget
+
+## Summary
+
+Machine disruption budget controller should monitor machines and **MachineDisruptionBudget** objects and updates accordingly the **MachineDisruptionBudget** object status.
+It will provide the updated ingormation regarding the number of healthy machines with specific labels under the cluster that can be used by another controllers.
```
Comment (enxebre) [inline docs/machine-disruption-budget-kpe.md]: typo s/ingormation/information

**Comment by enxebre** [comment]
Thanks! Overall lgtm to start with a feature gated implementation. Can we move this under `docs/proposals`?
/approve

**Pull request merged**